src/control/cmd/dmg/pool_test.go

```diff
@@ -581,6 +585,35 @@ func TestPoolCommands(t *testing.T) {
 			}, " "),
 			nil,
 		},
+		{
+			"Query pool with UUID",
+			"pool query --pool 12345678-1234-1234-1234-1234567890ab",
```

Comment:
That was my approach at first, but my thought was that in that case we should probably change --pool to --uuid. Additionally, the implementation of the human-friendly resolver was designed to be expanded to other unique identifiers that we may add later.

I'm not wedded to this approach, but looking at it from a user's perspective rather than someone who's been using and developing DAOS for a couple of years, this actually seems more intuitive than having separate flags.

Reply:
Incidentally, I know that @jolivier23 has suggested that we remove the flag altogether and just accept the pool ID as a positional argument, e.g. `dmg pool query foo` or `dmg pool query 12345678-1234-1234-1234-1234567890ab`. I rather like that personally, but it's my understanding that the flags were requested for consistency with the daos tool.